pygls/types.py

```diff
@@ -230,7 +230,7 @@ def __init__(self,
                  sort_text: str = None,
                  filter_text: str = None,
                  insert_text: str = None,
-                 insert_text_format: str = None,
+                 insert_text_format: int = 1,
```

Comment:
Could you please replace `1` with `InsertTextFormat. PlainText` as a default value?

Reply:
How about to change just annotation type and leave default value unspecified? Please see updated patch.

Reply:
Sounds good.